src/server.js

```diff
@@ -35,7 +33,7 @@ class Server {
 
         //NOTE: getting data from github
         const apiGithubObj = new ApiGithub(token);
-        const listWFRuns   = await apiGithubObj.getListRuns(owner, repo);
+        const listWFRuns   = owner && repo ? await apiGithubObj.getListRuns(owner, repo) : [];
```

Comment:
It's works here, but you cannot just check this properties always and everywhere. You will forget this check in some place, and you app will behave strange and produce hard-to-catch bugs. 

For your app, `owner`, `repo` and `token` are *required* properties. It cannot function without them. So it is the responsibility of your `Config` class to check required properties and if some of them is missing - throw an exception and terminate your app.